test/controllers/util/DisplayPropertiesUtilsSpec.scala

```diff
@@ -53,17 +53,41 @@ class DisplayPropertiesUtilsSpec extends AnyFlatSpec with MockitoSugar with Befo
     dropdownField.fieldDescription should equal("description")
```

Comment:
Should probably rename this test to account for it not being multi-value: `"generate 'dropdown' field for componentType value 'select' where the property is not mult-value"`